ultet/src/main/scala/za/co/absa/ultet/dbitems/DBSchema.scala

```diff
@@ -16,17 +16,38 @@
 
 package za.co.absa.ultet.dbitems
 
+import com.typesafe.scalalogging.Logger
+import za.co.absa.ultet.dbitems.DBSchema.{DO_NOT_CHOWN, DO_NOT_TOUCH, logger}
 import za.co.absa.ultet.model.{SQLEntry, SchemaName, UserName}
 import za.co.absa.ultet.model.schema.{SchemaCreate, SchemaGrant, SchemaOwner}
 
 case class DBSchema(name: SchemaName,
                     ownerName: UserName,
                     users: Seq[UserName]) extends DBItem {
   override def sqlEntries: Seq[SQLEntry] = {
-    Seq(
-      SchemaCreate(name),
-      SchemaOwner(name, ownerName),
-      SchemaGrant(name, users)
-    )
+    if (DO_NOT_TOUCH.contains(name.value)) {
+      throw new Exception(s"Schema ${name.value} is not allow to be referenced")
+    }
+
+    if (DO_NOT_CHOWN.contains(name.value)){
+      logger.warn(s"Schema ${name.value} is not allowed to change owner")
```

Comment:
This will print a warning anytime we have something in `public` schema. But OK for now.